src/memex/resources.py

```diff
@@ -16,17 +16,19 @@ def __getitem__(self, id):
         annotation = storage.fetch_annotation(self.request.db, id)
         if annotation is None:
             raise KeyError()
-        return AnnotationResource(self.request, annotation)
+
+        group_service = self.request.find_service(IGroupService)
```

Comment:
Which was exactly the goal to make the [streamer code work](https://github.com/hypothesis/h/pull/4299/files#diff-23fa267bf8100c9a7401f260e0e347d4R150).